services/QuillLMS/engines/evidence/app/models/evidence/research/gen_ai/trial.rb

```diff
@@ -119,6 +122,13 @@ def set_trial_start_time = update!(trial_start_time: Time.zone.now)
         def set_trial_duration = update!(trial_duration: Time.zone.now - Time.zone.parse(trial_start_time))
         def set_status = trial_errors.empty? ? completed! : failed!
 
+        def set_evaluator_counts
+          evaluator_failure_count = feedback_evaluations.suboptimal.count
```

Comment:
It feels weird here to switch the language from 'suboptimal' to 'failure'.  Should consistent language be used throughout?

Reply:
That's fair, will rename.